src/lualib/declarations/coroutine.d.ts

```diff
@@ -0,0 +1,19 @@
+/** @noSelfInFile */
```

Comment:
Do we still need these lualib declarations now we included lua-types as dependency?

Reply:
lua-types is used only in benchmark project, not in lualib. It should be possible to use it in the future, but it probably would need some code changes, since some things are declared differently